trulens_eval/examples/snowflake_demo/llm.py

```diff
@@ -1,26 +1,34 @@
-import json
 import re
-from typing import AsyncIterator, List, Optional
+from typing import Any, AsyncIterator, List, Optional
 
+from feedback import f_small_local_models_context_relevance
+from feedback import get_provider
 import replicate
 from retrieve import AVAILABLE_RETRIEVERS
 from schema import Conversation
 from schema import Message
+from schema import ModelConfig
 import streamlit as st
 from streamlit.delta_generator import DeltaGenerator
 
+from trulens_eval.guardrails.base import context_filter
 # replicate key for running model
 from trulens_eval.tru_custom_app import instrument
-from trulens_eval.guardrails.base import context_filter
-
-from feedback import f_small_local_models_context_relevance
 
-FRIENDLY_MAPPING = {
-    "Snowflake Arctic": "snowflake/snowflake-arctic-instruct",
-    "LLaMa 3 8B": "meta/meta-llama-3-8b",
-    "Mistral 7B": "mistralai/mistral-7b-instruct-v0.2",
+PROVIDER_MODELS = {
+    "Replicate":
+        {
+            "Snowflake Arctic": "snowflake/snowflake-arctic-instruct",
+            "LLaMa 3 8B": "meta/meta-llama-3-8b",
+            "Mistral 7B": "mistralai/mistral-7b-instruct-v0.2",
```

Comment:
same here could we name this "Mistral 7B Instruct (v0.2)"?  Main reason I think of is to avoid user from confusing they are the same model (checkpoint)